spaces/diadata/index.json

```
@@ -6,11 +6,12 @@
   "skin": "diadata",
   "strategies": [
     {
-      "name": "erc20-balance-of",
+      "name": "erc20-balance-of-minimum",
```

`erc20-balance-of-minimum` is not an existing strategy. You could use an existing one https://github.com/snapshot-labs/snapshot.js/tree/master/src/strategies
```suggestion
      "name": "erc20-with-balance",
```